apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -158,32 +159,31 @@ public static void main(String[] args) {
         new Configuration[] {
           new WebInfConfiguration(),
           new WebXmlConfiguration(),
+          new WebAppConfiguration(),
           new MetaInfConfiguration(),
           new FragmentConfiguration(),
           new JettyWebXmlConfiguration(),
           new AnnotationConfiguration()
         });
 
     // Allow webapps to see but not override SLF4J (prevents LinkageErrors).
-    webapp.getSystemClasspathPattern().add("org.slf4j.");
-
+    webapp.addSystemClassMatcher(new ClassMatcher("org.slf4j."));
     /*
      * Disable Logback's builtin shutdown hook, so that OUR shutdown hook can still use the loggers
      * (and then shut Logback down itself).
      */
     webapp.setInitParameter("logbackDisableServletContainerInitializer", "true");
 
     /*
-     * Configure the NCSA-style flat access/request log. This picks up its configuration from this
-     * project's logback-access.xml resource. Please note that the logback-access library it uses is
-     * only half-supported anymore and will miss a number of things, e.g. async requests. See
-     * https://github.com/qos-ch/logback/pull/269 for details.
+     * Logback access does not seem to be supported any longer with jetty 10 so switching to
+     * a custom format request log. Need to evaluate if we still need this at all and fill in
+     * the format string to match what we had before or find a new approach entirely or
+     * somehow hack the logback-access to make it work again.
      */
     RequestLogHandler requestLogHandler = new RequestLogHandler();
-    JettyLogbackRequestLogImpl requestLog = new JettyLogbackRequestLogImpl();
-    requestLog.setName("access.log");
-    requestLog.setResource("/logback-access.xml");
-    requestLog.setQuiet(true);
+    final String accessLogFileName =
+        System.getProperty("bfdServer.logs.dir", "./target/server-work/") + "access.log";
+    CustomRequestLog requestLog = new CustomRequestLog(accessLogFileName);
```

Comment:
I added this but it could use some refinement -- we should try to replicate the data that is in the logback-access.xml (and that file should be removed as well). Also the comment can be adjusted at that time.

Reply:
Locally things are looking as they should. Will resolve this if deploying to the test server is showing any inconsistencies